src/arvgcboolean.c

```diff
@@ -213,6 +213,27 @@ arv_gc_boolean_set_value (ArvGcBoolean *gc_boolean, gboolean v_boolean, GError *
 		g_propagate_error (error, local_error);
 }
 
+static ArvGcAccessMode
+arv_gc_boolean_get_access_mode (ArvGcFeatureNode *gc_feature_node)
+{
+	ArvGcBoolean *gc_boolean = NULL;
+	ArvGcNode *pvalue_node = NULL;
+	ArvGcFeatureNode *pvalue_feature = NULL;
+
+	g_return_val_if_fail (ARV_IS_GC_BOOLEAN (gc_feature_node), ARV_GC_ACCESS_MODE_RO);
+	gc_boolean = ARV_GC_BOOLEAN (gc_feature_node);
+	if (gc_boolean->value == NULL)
+		return ARV_GC_ACCESS_MODE_RO;
+
+	pvalue_node = arv_gc_property_node_get_linked_node (gc_boolean->value);
+	g_return_val_if_fail (ARV_IS_GC_FEATURE_NODE (pvalue_node), ARV_GC_ACCESS_MODE_RO);
+
+	pvalue_feature = ARV_GC_FEATURE_NODE (pvalue_node);
```

Comment:
Virtual should not be called directly, usually there is a function defined in the class declaring the virtual that does that.